gulpfile.js

```diff
@@ -169,6 +170,7 @@ createTask('prettify', prettify);
 createTask('process-3p-github-pr', process3pGithubPr);
 createTask('process-github-issues', processGithubIssues);
 createTask('release', release);
+createTask('report-test-results', reportTestResults);
```

Comment:
nit: can we rename this to something-something-upload like `codecov-upload`?

Reply:
Done.